basic-spring-boot/requirements.yml

```diff
@@ -0,0 +1,8 @@
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+# to support the operation of CASL provisioning/runs.
+
+# From 'openshift-applier'
+- name: openshift-applier
+  scm: git
+  src: https://github.com/redhat-cop/openshift-applier
+  version: v2.0.2
```

Comment:
Since v2.0.3 has been out for a couple of weeks, I'd recommend going to that version. 

Also, if you are targeting remote hosts, I'd recommend waiting on v2.0.4 (or submit a follow-up PR) as it will contain this PR: https://github.com/redhat-cop/openshift-applier/pull/68

Reply:
@oybed not planning on targeting remote hosts. updated to 2.0.3